CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+# Contribution Guide
+
+We're happy to accept contributions from the community. To do so, simply submit
+a pull request to this repository.
+
+## Contribution Checklist
+
+### Pull Request Scope
+
+We prefer the scope of each change to be as limited as possible. In most cases
+that means only a single "module" should be modified in one pull request,
+especially if other modules depend on it. In this context, a "module" is a
+subdirectory containing a Dockerfile, such as `monasca-agent`,
+`monasca-api-python`, `mysql-init`, and so on.
+
+That said, it occasionally makes sense to group a number of small changes
+together into one pull request, such as when adding a new linter rule. A hard
+maximum of 5 modules can be modified in a single pull request, however.
+
+### Releasing Changes
+
+By default, we do not publish new images with every change. An image push is
+triggered in one of two ways:
+
+ * The build configuration (`build.yml`) is changed, usually to increment a
+   version number
+ * A `!push <module>` tag is included in the merge commit message
+
+Several components are not given explicit versions as they track upstream Git
+repositories, so tags are generated automatically at build time. As these
+automatic
```

Comment:
rest of the sentence got eaten

Reply:
Good catch, I'll make sure to fix it. Thanks!